client/src/lib/components/modals/UserModal.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import { closeModal } from 'svelte-modals';
+
+	// provided by Modals
+	export let isOpen: boolean;
+</script>
+
+{#if isOpen}
+	<div class="modal">
```

Comment:
Trying to move to more utility components like this when there is a repeated, wordy piece of html (in lieu of custom css classes, e.g.)